native/SampleApps/SmartSyncExplorer/SmartSyncExplorer/Classes/ContactListViewController.m

```diff
@@ -83,7 +84,7 @@ @implementation ContactListViewController
 - (id)initWithNibName:(NSString *)nibNameOrNil bundle:(NSBundle *)nibBundleOrNil {
     self = [super initWithNibName:nibNameOrNil bundle:nibBundleOrNil];
     if (self) {
-        self.dataMgr = [[SObjectDataManager alloc] initWithViewController:self dataSpec:[ContactSObjectData dataSpec]];
```

Comment:
Data manager no longer has a reference to the parent view - instead, on refresh it expects a block